Update passwordvault_add_615680936.md

pranavkukreja opened the pull request.

pranavkukreja committed:
Update passwordvault_add_615680936.md

pranavkukreja commented:
Default is 20 credentials

mikematteson commented:
@PatrickFarley : PR **passed** acceptance checklist review. Please evaluate further and approve/merge if appropriate. Thanks!

PatrickFarley committed:
formatting

Generally it's ok to write numbers 1-10 as words, but above that we use numerals.
